docs/pages/config-plugins/development-and-debugging.mdx

```diff
@@ -637,16 +636,16 @@ This does come with a couple of caveats:
 3. Plugins can only be automatically added when the user's project uses a static app config (**app.json** and **app.config.json**).
    If the user runs `expo install expo-camera` in a project with an **app.config.js**, they'll see a warning like:
 
-  ```text
-  Cannot automatically write to dynamic config at: app.config.js
-  Please add the following to your app config
+```text
+Cannot automatically write to dynamic config at: app.config.js
+Please add the following to your app config
 
-  {
-    "plugins": [
-      "expo-camera"
-    ]
-  }
-  ```
+{
+  "plugins": [
+    "expo-camera"
+  ]
+}
+```
```

Comment:
This was indented on purpose, so the code block appears nested under `3.` list element.

![Screenshot 2024-06-30 083050](https://github.com/expo/expo/assets/719641/4f605b81-4082-44a9-a352-7e67db142ca4)

Reply:
If that's the case, I'll revert it and add prettier-ignore rule.

Reply:
I've reverted this change and added the prettier-ignore comment for this code snippet block. FYI, this doesn't have an effect on user-facing docs (whether we change it or not):

![CleanShot 2024-06-30 at 18 44 18@2x](https://github.com/expo/expo/assets/10234615/9df04d5a-73ac-48ad-b802-fc0c86781b31)

Link to the section: https://docs.expo.dev/config-plugins/development-and-debugging/#expo-install

Reply:
We are missing one more space per line.

Reply:
I overlooked that, my bad. Let me fix that!

Reply:
Fixed it (also thanks for the tip to use shell code language.

![CleanShot 2024-06-30 at 23 53 37](https://github.com/expo/expo/assets/10234615/0ac70529-591d-4a18-9142-0c8bd69837bc)